src/commands/releases.rs

```diff
@@ -507,6 +517,63 @@ fn execute_set_commits<'a>(
     Ok(())
 }
 
+fn execute_set_manual_commits<'a>(ctx: &ReleaseContext<'_>, matches: &ArgMatches<'a>) -> Result<(), Error> {
+    let version = matches.value_of("version").unwrap();
+    let org = ctx.get_org()?;
+
+    if let Ok(projects) = ctx.get_projects(matches) {
+        ctx.api.new_release(
+            &org,
+            &NewRelease {
+                version: version.into(),
+                projects,
+                ..Default::default()
+            },
+        )?;
+    }
+
+    // Get the commit of the most recent release.
+    let prev_commit = match ctx.api.get_previous_release_with_commits(org, version)? {
+        OptionalReleaseInfo::Some(prev) => prev.last_commit.map(|c| c.id).unwrap_or_default(),
+        OptionalReleaseInfo::None{} => String::new(),
+    };
+
+    // Find and connect to local git.
+    let repo = git2::Repository::open_from_env()?;
+
+    // Parse the git url.
+    let remote = repo.find_remote("origin")?;
```

Comment:
Use `config.get_cached_vcs_remote()` instead. This way you'll get free remote configuration feature.